lib/auth/auth_with_roles.go

```diff
@@ -1518,7 +1517,7 @@ func (a *ServerWithRoles) ListUnifiedResources(ctx context.Context, req *proto.L
 			"elapsed_filter": elapsedFilter,
 		}).Debugf(
 			"ListUnifiedResources(%v->%v) in %v.",
-			len(unifiedResources), len(filteredResources), elapsedFetch+elapsedFilter)
+			len(unifiedResources), len(unifiedResources), elapsedFetch+elapsedFilter)
```

Comment:
looks like this logging is deferred until the end, wouldn't `ListUnifiedResources(%v->%v)` be the same? b/c they both refer to the same resources, should it be `0 -> len(unifiedResources)`?